.github/workflows/es-actions.yml

```diff
@@ -174,6 +174,61 @@ jobs:
           CC=clang CXX=clang++ cmake -H. -Bout/release -DESCARGOT_ARCH=aarch64 -DESCARGOT_HOST=linux -DESCARGOT_MODE=release -DESCARGOT_TEMPORAL=ON -DESCARGOT_TCO=ON -DESCARGOT_TEST=ON -DESCARGOT_OUTPUT=shell -GNinja
           ninja -Cout/release
           python3 ./tools/run-tests.py --engine="./out/release/escargot" new-es
+  test-on-windows-clang-cl:
+    runs-on: windows-2022
+    strategy:
+      matrix:
+        # clang-cl with cannot generate c++ exception code well
+        # if clang-cl bug fixed, we can add x64
+        arch: [
+          {cpu: "x86", flag: "-m32"}
```

Comment:
The exception bug of clang-cl exists only for x64 target?

Reply:
Unfortunately no.. :(
1. x86 cannot run every test262 tests
2. x64 cannot run full octane tests